### DISCUSSION TITLE
Robots cannot be installed on Azerothcore

### DISCUSSION THREAD

**Discussion by zjgzh001** [trickerer/Trinity-Bots#461] [Q&A]
The following error occurred when entering this command git apply npcbots.patch in cmd. Please provide guidance
E:\azbotcoreserever\azerothcore-wotlk>git apply npcbots.patch
npcbots.patch:1542: trailing whitespace.
('70541', 'ruRU', 'Снять бафф', '', NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL),
npcbots.patch:9926: trailing whitespace.
npcbots.patch:11872: space before tab in indent.
                LOG_ERROR("entities.player", "NPCBots: bot text string #{} is not localized, at least for {}",
npcbots.patch:25805: space before tab in indent.
                stmt->SetData(i, npcBotData->equips[i]);
npcbots.patch:29159: space before tab in indent.
        mapid = data->mapid;
error: patch failed: src/server/game/Entities/Unit/Unit.cpp:1458
error: src/server/game/Entities/Unit/Unit.cpp: patch does not apply

**Comment by trickerer** [comment]
Wait until weekly update (saturday).
You can also try `patch -p1 < NPCBots.patch`

**Reply by zjgzh001** [reply]
Okay, thank you for your reply

**Reply by zjgzh001** [reply]
How to use this command under Windows: patch - p1<NPCBots. patch

**Reply by trickerer** [reply]
Right click in AC root folder -> 'Git Bash here'. This will give you MinGW terminal where this command is available.

**Reply by zjgzh001** [reply]
Thank you for your guidance. Thank you
